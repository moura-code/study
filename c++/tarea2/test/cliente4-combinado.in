crearCliente 31 Filemon Gonzalez 29
imprimirCliente
imprimirEdadCliente
imprimirIdCliente
copiarCliente
imprimirCliente
liberarCliente
Fin
