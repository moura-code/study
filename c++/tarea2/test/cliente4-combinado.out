Bienvenido al programa principal de Mercado FINGer. Por favor ingrese su comando:
1> Cliente creado de forma exitosa.
2> Cliente Filemon Gonzalez
Id: 31
Edad: 29
3> La edad del cliente es: 29
4> El id del cliente es: 31
5> Cliente copiado. Datos de la copia:
Cliente Filemon Gonzalez
Id: 31
Edad: 29
6> Cliente Filemon Gonzalez
Id: 31
Edad: 29
7> Cliente liberado con exito.
8> Fin.
